client/src/containers/backend/Permission/List.js

```diff
@@ -57,7 +57,7 @@ export class PermissionContainer extends PureComponent {
               }}
             />
           ) : null}
-          <HigherOrder.Authorize entity={entity} ability="updatePermissions">
+          <HigherOrder.Authorize entity={entity} ability="createPermissions">
```

Comment:
I wonder if we should wrap the List.SimpleList above with a check on manage permissions for the entity. Thoughts?

Reply:
The permissions container has a `managePermissions` ability check on it.  I think this is the right approach because it is validating a user has permission to see anything under the project permissions path.  This includes the list, but also edit, etc.  To me, I think think it makes sense to keep the "add" button check separate, but if not, it could be removed entirely as you cannot see the List without manage permissions.

Reply:
https://github.com/ManifoldScholar/manifold/pull/1078/files#diff-a7c704a3d4c0102626ad9dece07eab24R23